Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGalleryCell.swift

```diff
@@ -66,32 +65,48 @@ struct BasemapGalleryCell: View {
         .disabled(item.isBasemapLoading)
     }
     
+    /// Creates a badge which indicates the basemap supports 3D visualization.
+    /// - Returns: A 3D badge overlay view.
+    @ViewBuilder private func make3DBadge() -> some View {
+        if item.is3D {
+            Text(
+                "3D",
+                bundle: .toolkitModule,
+                comment: "A reference to 3D content."
+            )
+            .font(.basemapGallery)
+            .foregroundStyle(item.hasError ? .secondary : .primary)
+            .padding(2.5)
+            .background(Material.regular)
+            .cornerRadius(5)
+            .padding(2.5)
+        }
+    }
+    
     /// Creates an overlay which is either a selection outline or an error icon.
     /// - Returns: A thumbnail overlay view.
-    private func makeOverlay() -> some View {
-        Group {
-            if item.hasError {
-                HStack {
-                    Spacer()
-                    VStack {
-                        ZStack {
-                            // For a white background behind the exclamation mark.
-                            Circle()
-                                .foregroundStyle(.white)
-                                .frame(width: 16, height: 16)
-                            Image(systemName: "exclamationmark.circle.fill")
-                                .foregroundStyle(.red)
-                                .frame(width: 32, height: 32)
-                        }
-                        Spacer()
+    @ViewBuilder private func makeOverlay() -> some View {
+        if item.hasError {
```

Comment:
No changes here, apart from removing the `Group` and marking the method `@ViewBuilder`.